com.microsoft.mrtk.uxcore/Interop/TestInteractor.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Microsoft.MixedReality.Toolkit.UX
+{
+    public class InternalTestInteractor: IXRInteractor
```

Comment:
I don't believe this is supposed to be here?

Reply:
(It also generates warnings and is missing the copyright notice, so I figured this was just leftover)

Reply:
snuck in with the upstream merge, removing